WordPress/openverse
# Update .eslintrc.js file

Fixes #[[3386](https://github.com/WordPress/openverse/issues/3386)] by @[bhavik001 ]

## Description
This PR incorporates changes to the ESLint configuration to enforce the use of braces in one-line if-else statements and case blocks inside switch statements. This update aims to enhance code readability and prevent potential issues in the future. The specific ESLint rules added are:

- curly: Enforce braces for all control statements.
- default-case: Enforce a default case in switch statements.
- eqeqeq: Enforce the use of strict equality (===) and inequality (!==).

For more details on ESLint rules, please refer to the [official ESLint documentation](https://eslint.org/docs/rules).


## Testing Instructions

1. Ensure ESLint is installed by running npm install eslint --save-dev.
2. Run ESLint on the project to check for adherence to the new rules: npx eslint .
3. If necessary, automatically fix fixable issues using: npx eslint --fix .


## Checklist


- [x] I added or updated documentation regarding the ESLint rule changes.
- [x] I ran the project locally and verified that there are no visible errors.
